example_sdk_client/example_sdk_client_simulator.py

```diff
@@ -23,6 +23,8 @@ def handle_on_finished(job: Job, result: JobResult):
     )
 
     print(result.logs)
+    print("--------------")
```

Comment:
I would change the order of the output: first the logs, then the output esdl, and in the end the currently first print statement. This saves scrolling to the top for the overview.

Reply:
Is this what you meant? https://github.com/Project-OMOTES/computation-engine/pull/45/commits/0ffe890f08663a96bf86b8a6949a3f32fb63305f